webpack.config.js

```diff
@@ -16,6 +16,7 @@ module.exports = (_, env) => {
       'app': path.resolve(__dirname, 'src/app/main.ts'),
       'diagnostics': path.resolve(__dirname, 'src/diagnostics/main.js'),
       'feedback': path.resolve(__dirname, 'src/feedback/loader.js'),
+      'html2canvas': path.resolve(__dirname, `node_modules/html2canvas/dist/${html2canvas}`),
```

Comment:
Note-to-self: `html2canvas` is restored as a `webpack`-managed entrypoint -- instead of simply a file that is copied into the `public` directory for copy into the container image -- so that the `SubresourceIntegrityPlugin` emits an integrity hash for it, to be included dynamically in the CSP `script-src` hash list.